website/templates/project/settings.mako

```diff
@@ -77,24 +77,24 @@
                     <div id="projectSettings" class="panel-body">
                         <div class="form-group">
                             <label>Category:</label>
-                            <select data-bind="attr.disabled: disabled,
-                                                         options: categoryOptions,
-                                                         optionsValue: 'value',
-                                                         optionsText: 'label',
-                                                         value: selectedCategory"></select>
```

Comment:
On second thought, I would restore this to how it was before with everything in one line.